src/octopeer-github/content/ContentController.ts

```diff
@@ -18,6 +18,7 @@ class ContentController {
      * @return this
      */
     public start() {
+        Options.init();
```

Comment:
So why wasn't this already initialized here (before this PR)? :P 
Because of the DoNotWatchOptions?